docs/sources/alerting/unified-alerting/high-availability/enable-alerting-ha.md

```diff
@@ -0,0 +1,32 @@
++++
+title = " Enable alerting high availability"
+description = "Enable alerting high availability"
+keywords = ["grafana", "alerting", "tutorials", "ha", "high availability"]
+weight = 450
++++
+
+# Enable alerting high availability
+
+You can [enable alerting high availability]({{< relref "./_index.md" >}}) support by updating the Grafana configuration file. On Kubernetes, you can enable alerting high availability by updating the Kubernetes container definition.
+
+## Update Grafana configuration file
+
+Gossiping of notifications and silences uses both TCP and UDP port `9094`. Therefore, each Grafana instance will need to be able to accept incoming connections on these ports.
+
```

Comment:
The Before you begin section is missing. Looks like you have identified a prerequisite in the Therefore sentence above.

Reply:
Done